backend/eventloop/application_event_loop/application_event_runner_deployments.go

```diff
@@ -457,6 +468,8 @@ func (a applicationEventLoopRunner_Action) cleanOldGitOpsDeploymentEntry(ctx con
 	} else if rowsDeleted == 0 {
 		// Log the error, but continue
 		log.V(sharedutil.LogLevel_Warn).Error(nil, "unexpected number of rows deleted for application", "rowsDeleted", rowsDeleted, "appId", deplToAppMapping.Application_id)
+	} else {
```

Comment:
We've already got `log.Info("deleting database Application, id: " + deplToAppMapping.Application_id)` a few lines above in this file, so probably don't need to also include this line, unless you think it has a value add.